src/todo.js

```diff
@@ -0,0 +1,105 @@
+import trashImgSrc from '../assets/trash-can-regular.svg';
+
+export default class TodoTasks {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+    this.taskList = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+    AddTodo = () => {
+      const { description } = this;
+      const { completed } = this;
+      const { index } = this;
+      this.taskList.push({ description, completed, index });
+      localStorage.setItem('tasks', JSON.stringify(this.taskList));
+    }
+
+    showTodoList = () => {
+      this.taskList.sort((a, b) => a.index - b.index);
+
+      const list = document.querySelector('.list');
+      list.innerHTML = '';
+
+      for (let i = 0; i < this.taskList.length; i += 1) {
+        const listItem = document.createElement('li');
+        listItem.className = 'hbb';
+        listItem.classList.add('added-task');
+        listItem.id = `${i}`;
+        listItem.innerHTML = `<input type="checkbox" class="check${i}" name="" id="" value="">
+            <input type="text" class="content content${i}" name="" id="${i}" value ="${this.taskList[i].description} [${this.taskList[i].completed}][${this.taskList[i].index}]">
```

Comment:
- Ouch! this is really close but one of the major requirements of this project is to build a website that displays a list that looks and behaves like the part of the minimalist project captured in the video below. 
- [ ] But I noticed in your project once I add any task the `index` & `completed` properties are added along with the `description` in the Interface, this is cluttering up the UI and doesn't behave as shown in the video below. 👇
    [![Minimalist example](https://img.youtube.com/vi/AcUd-_Yjjqg/0.jpg)](https://www.youtube.com/watch?v=AcUd-_Yjjqg)
- This is how your interface currently behaves when I add tasks: 👇
  ![firefox_Y6D93RUDsl](https://github.com/porag-m06/Porag-s-Amazing-To-Do-List/assets/104968236/3a5f435c-ff44-4fc5-8338-19e530e57579)